AAShareBubbles/AAShareBubbles.m

```diff
@@ -61,6 +61,11 @@ - (id)initWithPoint:(CGPoint)point radius:(int)radiusValue inView:(UIView *)inVi
         self.instagramBackgroundColorRGB = 0x2e5e89;
         self.favoriteBackgroundColorRGB = 0xedd013;
         self.whatsappBackgroundColorRGB = 0x00B000;
+        self.QQBackgroundColorRGB = 0x1A87DA;
```

Comment:
@mrfeiii  By conversion, let's use lowercase first letters. "qqBackgroundColorRGB" would be nice.